crates/core_arch/src/x86/sse.rs

```diff
@@ -1251,6 +1251,19 @@ pub unsafe fn _mm_loadr_ps(p: *const f32) -> __m128 {
     simd_shuffle4(a, a, [3, 2, 1, 0])
 }
 
+/// Loads unaligned 64-bits of integer data from memory into new vector.
+///
+/// `mem_addr` does not need to be aligned on any particular boundary.
+///
+/// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_loadu_si64)
+#[inline]
+#[target_feature(enable = "sse")]
```

Comment:
This should be `mmx` instead of `sse`.

Reply:
Shouldn't this be `sse`? Intel Intrinsics Guide https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_loadu_si64 shows that it is green (SSE) and not yellow (MMX).

Reply:
Oh wait nevermind, that's not the issue.